x-pack/plugins/osquery/public/live_queries/form/index.tsx

```diff
@@ -422,7 +411,7 @@ const LiveQueryFormComponent: React.FC<LiveQueryFormProps> = ({
       singleQueryDetails?.action_id ? (
         <ResultTabs
           actionId={singleQueryDetails?.action_id}
-          ecsMapping={serializedFormData.ecs_mapping}
+          ecsMapping={convertECSMappingToObject(ecsMapping)}
```

Comment:
I think it would be safer to use a serializer rather than using `convertECSMappingToObject` directly

Reply:
👍